README.md

```diff
@@ -3,18 +3,40 @@
 [![License badge](https://img.shields.io/badge/license-GPL-blue.svg)](https://opensource.org/licenses/GPL-3.0)
 [![Docker badge](https://img.shields.io/docker/pulls/dojot/iotagent-json.svg)](https://hub.docker.com/r/dojot/device-manager/)
 
+The DeviceManager handles all operations related to creation, , update and deletion of devices in dojot. For more information
+on that, check [this file](./docs/concepts.rst).retrieval
 
-This implements a service to manage devices and their configuration
+## Dependencies
+
+DeviceManager has the following dependencies:
+
+- flask (including flask_sqlalchemy)
+- psycopg2
+- marshmallow
+- requests
+- gunicorn
+- gevent
+- json-logging-py
+- kakfa-python
+
+But you won't need to worry about installing any of these - they are automatically installed when starting DeviceManager.
 
 ## How to run
+
+If you really need to run DeviceManager as a standalone process (without dojot's wonderful docker-compose), you can execute these commands:
+
 ```shell
 python setup.py develop
 gunicorn device-manager.app:app
 ```
 
 ## Using curl to manage devices
 
+In order to manage devices using DeviceManager, you can use curl (or Postman, if you prefer) to do that.
+DeviceManager listens to requests on port 5000. All endpoints are documented [here](./docs/apiary.apib)
+
 ### Creating device
```

Comment:
I guess we'd be best off removing this section? (it's heavily out-of-date)

Reply:
Sure!